index.html

```diff
@@ -51,27 +65,29 @@ <h4>i am a software developer</h4>
       <h1>My Recent Works</h1>
       <div class="img-container">
         <img src="./images/image1.png" alt="" />
-        <h2>Multi-Posts Stories</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul>
-          <li>
-            <a href="#">css</a>
-          </li>
-          <li>
-            <a href="#">html</a>
-          </li>
-          <li>
-            <a href="#">Bootstraps</a>
-          </li>
-          <li>
-            <a href="#">Ruby</a>
-          </li>
-        </ul>
-        <button>see project</button>
+        <div class="work-text">
+          <h2>Multi-Posts Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
```

Comment:
- [ ] [REQUIRED] Great implementation, but you need to stick to the design as much as possible (font). The font type you used for the `p` tag is different from the font used in the Figma.
- [ ] [REQUIRED] Kindly ensure that the work section cards are aligned to the center as it is in the figma template.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-06-09 12-52-08](https://user-images.githubusercontent.com/58692590/172844023-3de8c600-571d-4be1-ab12-0f27c3f0cd20.png) | ![Screenshot from 2022-06-09 13-11-45](https://user-images.githubusercontent.com/58692590/172844071-f45691a1-c15b-40d3-b85f-a8c434a715ef.png)